scripts/collect.sh

```diff
@@ -32,10 +32,18 @@ for desktop in `find $DESKTOP_DIR $DESKTOP_DIR2 -name '*.desktop' | LC_ALL=C sor
 
     package_name=`/bin/rpm -qf $desktop --qf %{NAME}`
 
-    eval $(grep "^X-SuSE-YaST-AutoInst" $desktop | sed -e 's/X-SuSE-YaST-/X_SuSE_YaST_/' )
+    X_SuSE_YaST_AutoInstSchema=`grep "^X-SuSE-YaST-AutoInstSchema" $desktop`
+    X_SuSE_YaST_AutoInstSchema=${X_SuSE_YaST_AutoInstSchema##*=}
```

Comment:
Yes, that's much better! What about making it an one-liner?

```shell
X_SuSE_YaST_AutoInstSchema=$(sed -n -e "s/^X-SuSE-YaST-AutoInstSchema[ \t]*=[ \t]*//p" "$desktop")
```

(Please test, I'm not a sed expert... :wink:)

Reply:
It does not really works robust. I have some problems with CR in some profiles. Could we please keep my way ?

Reply:
OK, everything is better then `eval()`.  :+1:
In the long term maybe we should rewrite that to Ruby... :thinking: